homeassistant/components/advantage_air/binary_sensor.py

```diff
@@ -0,0 +1,126 @@
+"""Binary Sensor platform for Advantage Air integration."""
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_MOTION,
+    DEVICE_CLASS_PROBLEM,
+    BinarySensorEntity,
+)
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
```

Comment:
DOMAIN should be exclusive to the platform domain itself (sensor)
```suggestion
from .const import DOMAIN as ADVANTAGE_AIR_DOMAIN
```

Reply:
Oh interesting, I've been using the pattern SENSOR_DOMAIN, not the other way around. I'll do some find and replace.

Reply:
I think it is still a valid requirement, I've been following it a couple of years